pkg/authorization/authorizer/scope/converter.go

```
@@ -80,10 +80,6 @@ func ScopesToVisibleNamespaces(scopes []string, clusterRoleGetter rbaclisters.Cl
 				break
 			}
 		}
-
-		if !found {
```

Hmm let us add something like `ignoreUnhandledScopes` as a `bool` param and let the caller tell us this is ok to ignore.

---

all callers will tell us that though ... can do it, but sounds a bit redundant

---

Seems bad to just drop it.  I keep wondering if it would just be better to let the callers ignore all errors.  Sigh, I cannot tell what would be best here.